_apps/remove-outside-collaborators.md

```diff
@@ -0,0 +1,78 @@
+---
+# A human-friendly name of your listing
+title: Remove Outside Collaborators
+# A short description of what your app does
+description: A GitHub Probot App that monitors and instantly removes any outside collaborators added to your Organization.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: remove-outside-collaborators
+# Include a few screenshots that show your app in action
+screenshots:
+- https://user-images.githubusercontent.com/7875028/44161123-6548c900-a08a-11e8-82c3-adc890d61ec0.png
```

Comment:
I'm not sure this screenshot demonstrates your app's behavior as well as the next one. Maybe you could swap the order or remove this one? Or take a screenshot that demonstrates the issues title?